azimuth_capi/tests/base.py

```diff
@@ -0,0 +1,5 @@
+from oslotest import base
```

Comment:
Out of curiosity: What does `oslotest` get us here that we don't get with `unittest`?

Reply:
good question, its related to the use of stestr... I should probably try drop that.

Reply:
done